shared/src/business/useCases/users/updateUserContactInformationInteractor.test.js

```diff
@@ -304,4 +305,56 @@ describe('updateUserContactInformationInteractor', () => {
       isUpdatingInformation: false,
     });
   });
+
+  it('should update the firmName if user is a practitioner and firmName is passed in', async () => {
+    await updateUserContactInformationInteractor(applicationContext, {
+      contactInfo,
+      firmName: 'testing',
+      userId: 'f7d90c05-f6cd-442c-a168-202db587f16f',
+    });
+    expect(
+      applicationContext.getPersistenceGateway().updateUser.mock.calls[0][0]
+        .user,
+    ).toMatchObject({
+      firmName: 'testing',
+    });
+  });
+
+  it('should return early if the firmName and contact info was not changed', async () => {
+    applicationContext
+      .getPersistenceGateway()
+      .getUserById.mockImplementation(() => ({
+        ...mockUser,
+        contact: contactInfo,
+      }));
+
+    await updateUserContactInformationInteractor(applicationContext, {
+      contactInfo,
+      firmName: mockUser.firmName,
+      userId: 'f7d90c05-f6cd-442c-a168-202db587f16f',
+    });
+
+    expect(
+      applicationContext.getPersistenceGateway().updateUser,
+    ).not.toBeCalled();
+  });
+
+  it('should return early if the firmName and contact info was not changed', async () => {
+    applicationContext
+      .getPersistenceGateway()
+      .getUserById.mockImplementation(() => ({
+        ...mockUser,
+        contact: contactInfo,
+      }));
+
+    await updateUserContactInformationInteractor(applicationContext, {
+      contactInfo,
+      firmName: mockUser.firmName,
+      userId: 'f7d90c05-f6cd-442c-a168-202db587f16f',
+    });
+
+    expect(
+      applicationContext.getPersistenceGateway().updateUser,
+    ).not.toBeCalled();
+  });
```

Comment:
Do my eyes deceive me, or is this a duplicate test?

Reply:
I think so, fixing now.